tests/move/test_move_core.py

```diff
@@ -501,6 +501,20 @@ def test_extra(self, mock_copy):
 
         mock_copy.assert_called_once_with(extra)
 
+    def test_album_with_tracks_and_extras_no_duplicate_copy(self, mock_copy):
+        """When album and its tracks/extras are in same batch, avoid duplicate copying.
+
+        See: https://github.com/orgs/MoeMusic/discussions/280
+        """
+        album = album_factory(num_tracks=2, num_extras=2)
+        mock_session = MagicMock()
+
+        items = [album] + list(album.tracks) + list(album.extras)
```

Comment:
tracks and extras are already lists so you don't need list() here

Reply:
Fixed!